tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -346,6 +346,243 @@ def test_end_date_gte_lte(self, url, expected_dag_run_ids, session):
         self.assertEqual(dag_run_ids, expected_dag_run_ids)
 
 
+class TestGetDagRunBatch(TestDagRunEndpoint):
+    @provide_session
+    def test_should_respond_200(self, session):
+        dag_runs = self._create_test_dag_run()
+        session.add_all(dag_runs)
+        session.commit()
```

Comment:
```suggestion

    def test_should_respond_200(self):
        dag_runs = self._create_test_dag_run()
        with create_session() as session:
            session.add_all(dag_runs)
```
WDYT?

Reply:
Or maybe it would be better to extend `_create_test_dag_run` with commit option? For example `_create_test_dag_run(commit=True)` so we can avoid code duplication

Reply:
Thanks @turbaszek . This is addressed in the latest commit.